app/views/master_files/hls_manifest.m3u8.erb

```diff
@@ -15,7 +15,12 @@ Unless required by applicable law or agreed to in writing, software distributed
 %>
 #EXTM3U
 <% if @master_file.has_captions? %>
-#EXT-X-MEDIA:TYPE=SUBTITLES,GROUP-ID="subs",LANGUAGE="<%= Settings.caption_default.language %>",NAME="<%= Settings.caption_default.name %>",AUTOSELECT=YES,URI="<%= caption_manifest_master_file_path %>"
+<% @master_file.supplemental_file_captions.append(@master_file.captions).each_with_index do |caption, index| %>
```

Comment:
It seems like order may be important here given the firs is marked as default below.  Does `DEFAULT` signify anything to the player?  If it does then I'd think we'd need to figure out a way for the user to select which captions is default from within the edit item workflow.

Reply:
`DEFAULT` is supposed to designate the caption chosen by the player without user input. I think making the change to the edit item workflow would probably be best as a follow-up ticket/PR?